### PR TITLE
JEN-958

### PR DESCRIPTION
**Pull request by abychko**
8.0: switch to -DWITH_SYSTEM=libs
[-] deleted every occurence of WITH_SSL from Jenkins GUI and pipeline
[-] replaced WITH_SSL with WITH_SYSTEM=libs in build scripts

### CODE REVIEW

**Comment by abychko** [comment]
I will post test results here when ready

**Review by laurynas-biveinis — COMMENTED** [review]
The script changes look OK, but this does need testing, in order to check what missing libs have to be installed

**Comment by abychko** [comment]
test ran: https://ps2.cd.percona.com/view/TEST/job/test-percona-server-8.0-pipeline/3/console

File: local/build-binary [hunk 1]
```diff
@@ -152,8 +151,8 @@ pushd ${WORKDIR}
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
         -DWITH_TOKUDB=${WITH_TOKUDB} \
         -DWITH_ROCKSDB=${WITH_ROCKSDB} \
-        -DWITH_SSL=${WITH_SSL} \
         -DWITH_MECAB=${WITH_MECAB} \
+        -DWITH_SYSTEM=libs \
```
Comment (laurynas-biveinis) [inline local/build-binary]: It's actually -DWITH_SYSTEM_LIBS=ON https://dev.mysql.com/doc/refman/8.0/en/source-configuration-options.html#option_cmake_with_system_libs

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: local/build-binary (hunk 1 above)
Comment (abychko) [reply local/build-binary]: deleted

**Comment by abychko** [comment]
new test run https://ps2.cd.percona.com/view/TEST/job/test-percona-server-8.0-pipeline/4/console

**Comment by laurynas-biveinis** [comment]
Please provide link to the whole job, once it completes

**Comment by abychko** [comment]
ok, will run whole matrix. makes sense, sorry

**Comment by abychko** [comment]
started matrix https://ps2.cd.percona.com/view/8.0/job/percona-server-8.0-valgrind-param/1/

**Comment by laurynas-biveinis** [comment]
@abychko please use 3637583bb5217f8e2a287870775fd5657de271bc, not HEAD, for any 8.0-testing jobs

**Comment by laurynas-biveinis** [comment]
If testing needs to be run now, ok to use HEAD

**Comment by abychko** [comment]
started https://ps2.cd.percona.com/job/percona-server-8.0-valgrind-param/4/

**Pull request closed without merging**